docs/help/account/billing.md

```diff
@@ -0,0 +1,44 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. Payment is by credit card only; we do not offer any alternatives at this time. We use Stripe to handle all payments, so we never store your bank details.
+
+## Invoices
+
+### Edit Your Billing Information
+
+You can edit the following billing information so that it appears on your upcoming invoices:
```

Comment:
It's more an addition than an edition, WDYT?

```suggestion
You can add the following billing information so that it appears on your upcoming invoices:
```

Reply:
IIRC, the billing information is added during the subscription in the subscription funnel. At least in the nominal use case, people can then edit what they entered previously.

Reply:
I'd stick to "edit" tbh, I feel aligned to Chris comment

Reply:
LGTM 👍